contrib/container/sources_ubuntu:22.04.list

```diff
@@ -0,0 +1,17 @@
+deb     [arch=amd64] http://archive.ubuntu.com/ubuntu jammy           main restricted universe multiverse
+deb-src [arch=amd64] http://archive.ubuntu.com/ubuntu jammy           main restricted universe multiverse
+deb     [arch=amd64] http://archive.ubuntu.com/ubuntu jammy-updates   main restricted universe multiverse
+deb-src [arch=amd64] http://archive.ubuntu.com/ubuntu jammy-updates   main restricted universe multiverse
+deb     [arch=amd64] http://archive.ubuntu.com/ubuntu jammy-backports main restricted universe multiverse
+deb-src [arch=amd64] http://archive.ubuntu.com/ubuntu jammy-backports main restricted universe multiverse
+deb     [arch=amd64] http://archive.ubuntu.com/ubuntu jammy-security  main restricted universe multiverse
+deb-src [arch=amd64] http://archive.ubuntu.com/ubuntu jammy-security  main restricted universe multiverse
+
+deb     [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy           main restricted universe multiverse
```

Comment:
`arm64` is in the main Ubuntu archive, so I think the separation is not necessary.
http://archive.ubuntu.com/ubuntu/dists/lunar/Release